openapi/definitions.yaml

```diff
@@ -367,6 +367,20 @@ MessageResponseWithContent:
       $ref: "#/MessageStatusValue"
   required:
     - message
+LegalMessageResponseWithContent:
+  type: object
+  properties:
+    message:
+      $ref: "#/CreatedMessageWithContent"
+    legal_message:
+      $ref: "#/LegalMessage"
+    notification:
+      $ref: "#/MessageResponseNotificationStatus"
+    status:
+      $ref: "#/MessageStatusValue"
+  required:
+    - message
+    - legal_message
```

Comment:
Can we use `allOf` with `MessageResponseWithContent`?

Reply:
refactored with allOf. Thank you!